jekyll/_cci2/deploy-service-update-to-aws-ec2.adoc

```diff
@@ -121,7 +121,7 @@ jobs:
           task-definition-arn: '${TASK_DEFINITION_ARN}'
 ----
 
-This section illustrates how you can use the orb to install and configure the AWS CLI, retrieve the [task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) that was previously deployed, and then _verify_ the revision has been deployed using the `verify-revision-is-deployed` command from the `AWS-ECS` orb.
+This section illustrates how you can use the orb to install and configure the AWS CLI, retrieve the link:https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html[task definition] that was previously deployed, and then _verify_ the revision has been deployed using the `verify-revision-is-deployed` command from the `AWS-ECS` orb.
```

Comment:
What is this syntax doing with the link:url[title]? I imagine it's adding the word "link:" before the link? Would there then need to be a space between "link:" and the definition name?

Reply:
sorry for the confusion here - it is the same as just `url[custom link text]` ... I won't do it in future, I was trying it out as it looks easier to see in a code editor but its not necessary.